week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -5,42 +5,59 @@
 */
 
 var person1 = {
-    name: "Alice",
-    age: 25
+  name: "Alice",
+  age: 25
 };
 
 var person2 = {
-    name: "Bob",
-    age: 30
+  name: "Bob",
+  age: 30
 };
 
 var person3 = {
-    name: "John",
-    age: 20
+  name: "John",
+  age: 20
 };
 
 /* 
 DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
+var persons = [person1, person2, person3];
+console.log(persons);
 
+var personNames = persons.map(namesOf => {
+  return namesOf.name;
+});
 
-var persons = // Complete here
-
-var personNames = // Complete here
-
-var personsYoungerThan28YearsOld = // Complete here
-
+var personsYoungerThan28YearsOld = persons.filter(ages => {
```

Comment:
`ages` is a misleading name for this parameter.  The parameter to the callback function given to persons.filter() should indicate its type.  The function doesn't get called with a list of ages.  What does it get called with?  What would be a better name?